ModularBungalow/clothing/coat.dm

```diff
@@ -99,7 +99,7 @@
 	inhand_icon_state = "armor"
 	dog_fashion = null
 
-/obj/item/clothing/suit/CoCoat
+/obj/item/clothing/suit/armor/centcom/CoCoat
```

Comment:
You do realize that it makes it as strong as death squad armor, if not more?
No. We certainly don't have to give some shitty janitor coat that much armor.

Reply:
Usually these Admin armor sets aren't seen.
Like ever. Even admins dont' use them during normal play

Reply:
Oh, so that's not available normally? If so - then it's fine.